backend/layers/persistence/orm.py

```diff
@@ -0,0 +1,82 @@
+from dataclasses import dataclass, field
+from typing import List
+import uuid
+from sqlalchemy import Column, DateTime, Enum, ForeignKey, String, Table
+from sqlalchemy.dialects.postgresql import ARRAY, BOOLEAN, JSON, TEXT, UUID
+from sqlalchemy.orm import registry
+from sqlalchemy.schema import MetaData
+
+from backend.layers.common.entities import DatasetArtifactType
+
+metadata = MetaData(schema="persistence_schema")
+mapper_registry = registry(metadata=metadata)
+
+
+@mapper_registry.mapped
+class Collection:
+
+    __table__ = Table(
```

Comment:
initially, these were inheriting the backend.layers.common.entity classes, and needed to be structured this way not to overwrite the class variables defined in the parent entity classes. Then, they ended up having more divergences than I anticipated so I removed that inheritance but didn't rewrite the table definition since this also works.

I don't have a strong opinion about one over the other but I can restructure if you think its worthwhile